### DISCUSSION TITLE
Extract, ignore an inforamtion

### DISCUSSION THREAD

**Discussion by ahmed-bhs** [eyurtsev/kor#236] [Q&A] [answered]
Is there a way to ignore or exclude results from being extracted from a text ?

**Comment by eyurtsev** [comment, marked as answer]
Follow the guidelines here: https://eyurtsev.github.io/kor/guidelines.html

If you still get undesirable results, you could try and do another pass with an LLM or else come up with some heuristics to remove false positives / incorrect extractions
